demos/face_recognition_demo/python/face_recognition_demo.py

```diff
@@ -228,7 +228,7 @@ def main():
             if frame_num == 0:
                 raise ValueError("Can't read an image from the input")
             break
-        if input_crop:
+        if input_crop.any():
```

Comment:
```suggestion
        if input_crop is not None:
```
I don't think `input_crop` can be an empty array. But `input_crop` can be `None`